ocean/sediment/sed_tracer/100m/config_forward2.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<config case="forward2">
+	<add_link source="../init/ocean.nc" dest="init.nc"/>
+	<add_link source="../init/mesh.nc" dest="mesh.nc"/>
+	<add_link source="../init/graph.info" dest="graph.info"/>
+	<add_link source="../gotmturb.nml" dest="gotmturb.nml"/>
+
+	<add_executable source="model" dest="ocean_model"/>
+
+	<namelist name="namelist.ocean" mode="forward">
+		<option name="config_ocean_run_mode">'forward'</option>
+		<option name="config_dt">'00:00:10'</option>
+		<option name="config_time_integrator">'RK4'</option>
+                <option name="config_run_duration">'0006_00:00:00'</option>
+		<option name="config_disable_tr_adv">.false.</option>
+		<option name="config_use_debugTracers">.true.</option>
+		<option name="config_use_sedimentTracers">.true.</option>
+                <option name="config_use_mom_del2">.true.</option>
+                <option name="config_mom_del2">1.0e-2</option>
+                <option name="config_use_tracer_del2">.true.</option>
+                <option name="config_tracer_del2">10.0</option>
+		<option name="config_use_cvmix">.false.</option>
+		<option name="config_use_gotm">.true.</option>
+		<option name="config_gotm_namelist_file">'gotmturb.nml'</option>
+		<option name="config_gotm_constant_bottom_roughness_length">5.0e-3</option>
+		<option name="config_gotm_constant_bottom_drag_coeff">1.73e-2</option>
+		<option name="config_use_implicit_bottom_drag">.true.</option>
+		<option name="config_implicit_bottom_drag_coeff">1.73e-2</option>
+	</namelist>
+
+	<streams name="streams.ocean" keep="immutable" mode="forward">
+		<stream name="mesh">
+			<attribute name="filename_template">mesh.nc</attribute>
+		</stream>
+		<stream name="input">
+			<attribute name="filename_template">init.nc</attribute>
+		</stream>
+		<stream name="output">
+                        <attribute name="type">output</attribute>
+<!---      
+      <attribute name="filename_interval">0001-00-00_00:00:00</attribute>
+      <attribute name="filename_template">output/output.$Y-$M-$D.nc</attribute>
+-->
```

Comment:
Please remove this or add a comment about under which circumstances it should be commented in.